### ISSUE TITLE
pre_train model

### ISSUE THREAD

**Issue by nightfuryyy** [xuexingyu24/License_Plate_Detection_Pytorch#3]
Can i use your model STN and LPRNet as init model. Thank you so much for awsome repo.

**Comment by nightfuryyy** [comment]
btw how can i change learning rate ? In sirius-ai repo, he use lr_schedule.

**Comment by xuexingyu24** [comment]
hi @nightfuryyy  definitely you could change learning rate as whatever you want. But I use Adam algorithm with which setting up learning rate is not quite common. While SGD with learning rate schedule is pretty much what people commonly do

**Comment by nightfuryyy** [comment]
thank you so much for helps.

**Comment by wait1988** [comment]
can you share your pretrained model for a quick  test? @xuexingyu24

**Comment by xuexingyu24** [comment]
@wait1988 

MTCNN : https://github.com/xuexingyu24/License_Plate_Detection_Pytorch/tree/master/MTCNN/weights

LPRNet: https://github.com/xuexingyu24/License_Plate_Detection_Pytorch/tree/master/LPRNet/weights
